soccer/gameplay/generate_fsm_diagrams.py

```diff
@@ -21,8 +20,20 @@ def mkdir_p(path):
 logging.getLogger().setLevel(logging.INFO)
 
 import main
+import ui.main
 main.init()
 
+class moc_Point():
```

Comment:
Hmm, I think you can use it since the `behavior-diagrams` makefile target depends on `all`, which ensures that the robocup python module is compiled beforehand. Also most of the plays/behaviors `import robocup` already. I think the catch here is that the robocup module has to be imported after the `sys.path.append('../../run')` line. Let me know if that doesn't work though.